src/nix/store-gc.md

```diff
@@ -8,14 +8,20 @@ R""(
   # nix store gc
   ```
 
-* Delete up to 1 gigabyte of garbage:
+* Perform garbage collection if there is less than 1 GiB of free space
+  in `/nix/store`, and stop once there is at least 5 GiB of free
```

Comment:
```suggestion
  on the file system partition holding `/nix/store`, and stop once there is at least 5 GiB of free
```

A directory doesn't have free space, right?

Reply:
Technically a file system can span many partitions (RAID, etc.) so I would just say "file system" and drop "partition".

Reply:
Right. Edited.

Reply:
> A directory doesn't have free space, right?

Well, a directory is always on a particular file system, so it can be said to have free space.

Reply:
```suggestion
  for `/nix/store`, and stop once there is at least 5 GiB of free
```
here is a lighter-weight alternative: "for" as in "for `/nix/store` to use". Exactly who or what is providing the free space, or how it is being provided, is left unstated as before.